### DISCUSSION TITLE
Resize breaks the shadow of animated webp

### DISCUSSION THREAD

**Discussion by avkabanov** [ImageMagick/ImageMagick#2926] [Development]
I have animated WebP with a transparent shadow so I need to resize it.
I do it with a command: `magick input.webp -layers coalesce -resize 200x200 -define webp:method=6 output.webp`
But when I open the resized image - I see that one frame has extremely thin bold, like all frames became visible at the same time. (example attached)

with another animation I've seen another result - with every new frame, the shadow becomes thicker like every new frame goes above the previous one but does not replace it. 

[example.zip](https://github.com/ImageMagick/ImageMagick/files/5643945/example.zip)

**Comment by avkabanov** [comment]
I've tried it on Big Sur 11.0.1 (20B29) with Chrome and Safari and still see the issue.

I've added HTML pages with color background so you can see the difference. Please open HTML here: 
[web_with_html.zip](https://github.com/ImageMagick/ImageMagick/files/5644238/1.zip)

Initially, I've tested and run `resize` on Windows 10, and opened the file with chrome, using HTML with a background color. 
```
$ magick --version
Version: ImageMagick 7.0.10-35 Q16 x64 2020-10-31 http://www.imagemagick.org
$ cwebp -version
1.0.0
```
`magick -list format` gives
```
     WEBP* WEBP      rw+   WebP Image Format (libwebp 1.0.3 [020E])
```

Here are two frames: 
all of the frames have the shadow like this: 
![image](https://user-images.githubusercontent.com/4610409/101193827-6b00b900-365d-11eb-8f97-e5b7c883795c.png)

But one frame has an extremely bold shadow: 
![image](https://user-images.githubusercontent.com/4610409/101193866-7653e480-365d-11eb-9f16-90312397ce33.png)

**Comment by fmw42** [comment]
When I view your images on my browsers, both input and output, I only see a black background, not transparent nor red. Apparently, my Chrome does not permit the transparency, so I am seeing black background. Same with Opera. So perhaps the Mac versions do not display the transparency correctly.  ImageMagick display does display transparency, but I cannot make it loop to get a better view. So perhaps I will leave this to the IM developers or someone else with better viewing capabilities.

**Reply by avkabanov** [reply]
please try open an HTML from a post above

**Reply by fmw42** [reply]
Try adding -dispose background

`magick -dispose background input.webp -layers coalesce -resize 200x200 -define webp:method=6 output.webp`

**Reply by avkabanov** [reply]
That gave even worse results, unfortunately  (sorry for the bad cropping). Now I always see only part of the picture
![image](https://user-images.githubusercontent.com/4610409/101198907-73102700-3664-11eb-85db-cc687b847c34.png)
![image](https://user-images.githubusercontent.com/4610409/101198973-8cb16e80-3664-11eb-92ce-3313c87fae94.png)
